.github/workflows/test-utils.yml

```diff
@@ -0,0 +1,101 @@
+name: "[test] utils"
```

Comment:
maybe `[test] misc` so that it won't get confused with utils package?

Reply:
Yes, makes sense. https://github.com/trezor/trezor-suite/pull/12716/commits/77718d8aa9a6058ae469be96222c879f942c097c